src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -49,11 +49,19 @@ internal class RestClientBuilder
         private readonly TypeFactory _typeFactory;
         private readonly Dictionary<string, Parameter> _parameters;
 
+        private Dictionary<InputOperation, Parameter> _bodyParameters;
```

Comment:
This is the wrong place for this change. We don't add any new `CreateXxxRequest` or protocol methods, only convenience methods will be added, so request methods shouldn't be affected.

Reply:
I got all the information for `ConvenienceMethods` from `ClientMethods`. But when we build `ClientMethods` by `RestClientBuilder`,  it throws away the important information like return type and body type. I just add its original information.